### PR TITLE
DOC: update automatic docs redirect

### PR DESCRIPTION
**Pull request by FredKSchott**
## Changes

- See title!
- adds support for `lang-region` codes

### CODE REVIEW

**Comment by Hanawa02** [comment]
LGTM!

**Review by jasikpark — APPROVED** [review]
Same here xD

File: docs/src/pages/index.astro [hunk 1]
```diff
@@ -6,13 +6,16 @@ import Layout from '../layouts/MainLayout.astro';
   // This is some wip redirect code based on the browser language.
   // A vercel.json redirect is enforced in production, so no user should ever see this page.
   // Remove the vercel.json redirect when this is ready.
-  const KNOWN_LANGUAGES = ['en', 'nl', 'fi','zh-CN','zh-TW'];
-  let newLang = (window.navigator.userLanguage || window.navigator.language || 'en').substr(0, 2);
+  const KNOWN_LANGUAGES = ['de','en','es','fi','nl','pt-br','zh-CN','zh-TW'];
```
Comment (jasikpark) [inline docs/src/pages/index.astro:9]: Would it be possible to construct a list of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl objects here instead? So we can have better verification that we're using valid codes + hopefully `newLangWithRegion` and `newLang` can just be function calls..

Comment (jasikpark) [reply docs/src/pages/index.astro:9]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/Locale a list of these

Comment (FredKSchott) [reply docs/src/pages/index.astro:9]: Yes! I'm not super familiar with this API but if you / someone wants to tackle in a follow up PR I'd love to review it!

**Pull request merged**